### PR TITLE
Update to Minecraft v1.19.20 (Protocol v544)

### PR DESCRIPTION
**Pull request by GoodLucky777**
Update to Minecraft v1.19.20 (Protocol v544) (#1392)

- Update resources (v1.19.20)
- Update protocol related things (v544)
- Update and fix block names

### CODE REVIEW

File: src/main/java/cn/nukkit/network/protocol/ModalFormResponsePacket.java [hunk 1]
```diff
@@ -16,7 +17,12 @@ public byte pid() {
     @Override
     public void decode() {
         this.formId = this.getVarInt();
-        this.data = this.getString(); //Data will be null if player close form without submit (by cross button or ESC)
+        if (this.getBoolean()) {
+            this.data = this.getString();
+        }
+        if (this.getBoolean()) {
+            this.cancelReason = this.getByte();
+        }
```
Comment (LoboMetalurgico) [inline src/main/java/cn/nukkit/network/protocol/ModalFormResponsePacket.java:25]:
```suggestion
        if (this.getBoolean()) {
            this.data = this.getString();
            this.cancelReason = this.getByte();
        }
```

Comment (LoboMetalurgico) [reply src/main/java/cn/nukkit/network/protocol/ModalFormResponsePacket.java:25]: Why not do this?

**Review by LoboMetalurgico — CHANGES_REQUESTED** [review]
Thanks for yet another incredible pull. I did a quick code review and found a small and simple issue.

I'll test later to see if I find any in-game issue.

File: src/main/java/cn/nukkit/network/protocol/ModalFormResponsePacket.java (hunk 1 above)
Comment (GoodLucky777) [reply src/main/java/cn/nukkit/network/protocol/ModalFormResponsePacket.java:25]: `BinaryStream.getBoolean()` increase `offset` of it (https://github.com/GoodLucky777/PowerNukkit/blob/a200820b41f7a6e81dde4e905c9df95d487cdcff/src/main/java/cn/nukkit/utils/BinaryStream.java#L224)

boolean - it is true if `data` exist
string - `data`
boolean - it is true if `cancelReason` exist
byte - `cancelReason`

https://github.com/CloudburstMC/Protocol/blob/0bd459f5dff7b32fca6b8049abe6ab93cea587af/bedrock/bedrock-v544/src/main/java/com/nukkitx/protocol/bedrock/v544/serializer/ModalFormResponseSerializer_v544.java#L32-L33

https://github.com/CloudburstMC/Protocol/blob/0bd459f5dff7b32fca6b8049abe6ab93cea587af/bedrock/bedrock-common/src/main/java/com/nukkitx/protocol/bedrock/BedrockPacketHelper.java#L798-L803

Comment (LoboMetalurgico) [reply src/main/java/cn/nukkit/network/protocol/ModalFormResponsePacket.java:25]: Oh, that's right.

**Review by LoboMetalurgico — APPROVED** [review]
Thank you again!

**Pull request merged**